Room Chating

shadabsaifi opened the pull request.

shadabsaifi committed:
Room Chating

shekhartyagi26 commented on server/server.js:
```diff
@@ -47,12 +48,20 @@ boot(app, __dirname, function(err) {
-      console.log('a user connected');
-      socket.on('disconnect', function(){
-          console.log('user disconnected');
+      // console.log('user connected');
```
why this is here?

shekhartyagi26 commented on server/server.js:
```diff
@@ -47,12 +48,20 @@ boot(app, __dirname, function(err) {
+        console.log("backend recieved msg===>>>>",msg)
+        app.io.emit('chat message', msg);
       });
+      // socket.on('disconnect', function(){
```
remove commented code.

shekhartyagi26 commented on server/server.js:
```diff
@@ -15,6 +15,7 @@ app.start = function() {
       var explorerPath = app.get('loopback-component-explorer').mountPath;
       console.log('Browse your REST API at %s%s', baseUrl, explorerPath);
     }
+   
```
where room is created.? please specify

shadabsaifi committed:
Readme.md

shadabsaifi committed:
Readme.md

shadabsaifi committed:
Readme.md

shadabsaifi committed:
Update README.md

shadabsaifi committed:
Update README.md

shadabsaifi committed:
Update README.md

shadabsaifi committed:
Readme.md

shadabsaifi committed:
Update README.md

shadabsaifi committed:
Update README.md

shadabsaifi committed:
Readme.md

shadabsaifi committed:
A

Merge branch 'develope' of https://github.com/shekhartyagi26/loopback into develope

shadabsaifi committed:
Readme.md

shadabsaifi committed:
Readme.md

shadabsaifi committed:
Update README.md

shadabsaifi committed:
Room Create

shadabsaifi committed:
update Readme.md

shadabsaifi committed:
remove commit code

shadabsaifi committed:
client code add

shadabsaifi committed:
Readme.md update

shadabsaifi committed:
Readme.md update

shadabsaifi committed:
Readme.md update

shadabsaifi committed:
Readme.md update

shekhartyagi26 committed:
Merge branch 'master' of https://github.com/shekhartyagi26/loopback into develope

shekhartyagi26 committed:
chat app aplication
